modules/social_features/social_event/src/Form/EnrollActionForm.php

```diff
@@ -199,7 +199,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $attributes = [
       'class' => [
         'btn',
-        'btn-accent brand-bg-accent',
+        'btn-default brand-bg-default',
```

Comment:
Just a FYI - i had to revert these changes as they gave issues in the default (no SKY) theme.